build.gradle

```diff
@@ -27,6 +27,7 @@ repositories {
 dependencies {
     implementation('org.springframework.boot:spring-boot-starter-data-jpa')
     implementation('org.springframework.boot:spring-boot-starter-web')
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
po co to?

Reply:
Bez tego nie zapisuje mi obiektów w testach:
![image](https://user-images.githubusercontent.com/114756001/225328878-72b9a951-5442-4ec9-8cf7-a4c994bfc2f8.png)